ws/portforward/model.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package portforward
+
+const (
+	MessageTypePortForwardNew  = "new"
+	MessageTypePortForwardStop = "stop"
```

Comment:
jsut something to consider for eventual debugging "new" will mean nothing, `new_forward` or anythig would be better.

Reply:
this is the equivalent of the `new` message of the shell protocol; I thought it makes sense to keep it consistent across the different protocols (`new` and `stop`). The ProtoType is what provides context to the message type.